pf9-autodeploy.yml

```diff
@@ -79,8 +91,11 @@
     - { role: "wait-for-convergence", when: autoreg == "on" }
     - { role: "k8s-cluster-attach", k8s_node_type: "worker", when: autoreg == "on" }
 
-# Run post_hook (if it exists)
-- hosts: all
+# Run post_hook
```

Comment:
shouldn't this run on all? I'd anticipate most folks have pre/post checks on every node, not only {hyper,container}visors.